src/System.Management.Automation/engine/BytePipe.cs

```diff
@@ -103,6 +104,10 @@ internal static FileBytePipe Create(string fileName, bool append)
             throw new RuntimeException(null, e, errorRecord);
         }
 
+        ApplicationInsightsTelemetry.SendExperimentalUseData(
+            ExperimentalFeature.PSNativeCommandPreserveBytePipe,
+            "native to file created");
```

Comment:
This can just be `File`
reduces the payload string by a bunch and still retains information.
I suppose it could also be `OutFile`